status-loop/queryValidators.js

```diff
@@ -126,6 +126,7 @@ async function getStatusOfCampaign(campaign) {
 	const statusName = getStatus(messagesFromAll, campaign, lastApprovedBalances)
 	return {
 		name: statusName,
+		endDate: campaign.status ? campaign.status.endDate : null,
```

Comment:
rename this (and the field in .status) to closedDate so that it's clear what it is

Reply:
ok